README.md

```diff
@@ -323,91 +194,99 @@ QEMU gets upset when you spawn multiple threads. This means that, if one of your
-| `thumbv8m.base-none-eabi` [4]        | 3.3.0  | 9.2.1   |     | N/A   |       |
-| `thumbv8m.main-none-eabi` [4]        | 3.3.0  | 9.2.1   |     | N/A   |       |
-| `thumbv8m.main-none-eabihf` [4]      | 3.3.0  | 9.2.1   |     | N/A   |       |
-| `wasm32-unknown-emscripten` [6]        | 3.1.14 | 15.0.0  | ✓   | N/A   |   ✓    |
-| `x86_64-linux-android` [1]           | 9.0.8  | 9.0.8   | ✓   | 6.1.0 |   ✓    |
-| `x86_64-pc-windows-gnu`              | N/A    | 9.3     | ✓   | N/A   |   ✓    |
-| `x86_64-sun-solaris`                 | 1.22.7 | 8.4.0   | ✓   | N/A   |       |
-| `x86_64-unknown-freebsd`             | 1.5    | 6.4.0   | ✓   | N/A   |       |
-| `x86_64-unknown-dragonfly` [2] [3]   | 6.0.1  | 10.3.0  | ✓   | N/A   |       |
-| `x86_64-unknown-illumos`             | 1.20.4 | 8.4.0   | ✓   | N/A   |       |
-| `x86_64-unknown-linux-gnu`           | 2.31   | 9.4.0   | ✓   | 6.1.0 |   ✓    |
-| `x86_64-unknown-linux-gnu:centos` [5]  | 2.17   | 4.8.5   | ✓   | 4.2.1 |   ✓    |
-| `x86_64-unknown-linux-musl`          | 1.2.3 | 9.2.0   | ✓   | N/A   |   ✓    |
-| `x86_64-unknown-netbsd` [3]          | 9.2.0  | 9.4.0   | ✓   | N/A   |       |
+| Target                                 |  libc  |  GCC   | C++ | QEMU  | `test` |
+|----------------------------------------|-------:|-------:|:---:|------:|:------:|
+| `aarch64-linux-android` [1]            | 9.0.8  | 9.0.8  | ✓   | 6.1.0 |   ✓    |
+| `aarch64-unknown-linux-gnu`            | 2.31   | 9.4.0  | ✓   | 6.1.0 |   ✓    |
+| `aarch64-unknown-linux-gnu:centos` [7] | 2.17   | 4.8.5  |     | 4.2.1 |   ✓    |
+| `aarch64-unknown-linux-musl`           | 1.2.3  | 9.2.0  | ✓   | 6.1.0 |   ✓    |
+| `arm-linux-androideabi` [1]            | 9.0.8  | 9.0.8  | ✓   | 6.1.0 |   ✓    |
+| `arm-unknown-linux-gnueabi`            | 2.31   | 9.4.0  | ✓   | 6.1.0 |   ✓    |
+| `arm-unknown-linux-gnueabihf`          | 2.31   | 8.5.0  | ✓   | 6.1.0 |   ✓    |
+| `arm-unknown-linux-musleabi`           | 1.2.3  | 9.2.0  | ✓   | 6.1.0 |   ✓    |
+| `arm-unknown-linux-musleabihf`         | 1.2.3  | 9.2.0  | ✓   | 6.1.0 |   ✓    |
+| `armv5te-unknown-linux-gnueabi`        | 2.31   | 9.4.0  | ✓   | 6.1.0 |   ✓    |
+| `armv5te-unknown-linux-musleabi`       | 1.2.3  | 9.2.0  | ✓   | 6.1.0 |   ✓    |
+| `armv7-linux-androideabi` [1]          | 9.0.8  | 9.0.8  | ✓   | 6.1.0 |   ✓    |
+| `armv7-unknown-linux-gnueabi`          | 2.31   | 9.4.0  | ✓   | 6.1.0 |   ✓    |
+| `armv7-unknown-linux-gnueabihf`        | 2.31   | 9.4.0  | ✓   | 6.1.0 |   ✓    |
+| `armv7-unknown-linux-musleabi`         | 1.2.3  | 9.2.0  | ✓   | 6.1.0 |   ✓    |
+| `armv7-unknown-linux-musleabihf`       | 1.2.3  | 9.2.0  | ✓   | 6.1.0 |   ✓    |
+| `i586-unknown-linux-gnu`               | 2.31   | 9.4.0  | ✓   | N/A   |   ✓    |
+| `i586-unknown-linux-musl`              | 1.2.3  | 9.2.0  | ✓   | N/A   |   ✓    |
+| `i686-unknown-freebsd`                 | 1.5    | 6.4.0  | ✓   | N/A   |        |
```

Comment:
the diff here looks wrong, specifically in regards to the whitespace in the C++ and test column

Reply:
I can't quite follow you there. To me the diff looks normal, there aren't any whitespace changes in the C++ column. I checked and the text doesn't contain literal tab chars either. Can you clarify what's off here?

Reply:
There is a space missing in c++ and test  when the cell does not contain a checkmark

Reply:
![Screenshot_20240213-075818](https://github.com/cross-rs/cross/assets/1502855/ddceba72-2a65-4cef-adbe-d7f76619491d)

Reply:
Actually, you're right, this is just a font issue where the checkmark is not mono